Aspekt.Bootstrap/IlGenerator.cs

```diff
@@ -10,7 +10,7 @@ namespace Aspekt.Bootstrap
     /// <summary>
     /// A static helper class for capturing and writing appropriate IL instructions
     /// </summary>
-    internal static class IlGenerator
+    public static class IlGenerator
```

Comment:
Why make this public? If it's for tests, I'd recommend using the InteralsVisibleTo assembly attribute instead (see AssemblyInfo.cs).

Reply:
Hmmmm. I'm not sure why I did this. Could've been a mistake I missed. (I should review better ;))